app/models/event.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# Event
+class Event < ApplicationRecord
+  has_many :attendance, foreign_key: :attended_event_id, inverse_of: :attendances, dependent: :destroy
```

Comment:
Can you explain further the idea behind using `inverse_of`?

Reply:
We included inverse_of because of the linter Rubocop which was requiring it. We'll revise this!